app/models/Screening.scala

```diff
@@ -0,0 +1,4 @@
+package models
+import com.github.nscala_time.time.Imports._
+
+case class Screening(id: Int, startDateTime: DateTime, /*startHour: Int, startMinute: Int, */finishDateTime: DateTime, /*finishHour : Int, finishMinute : Int, */movie: Movie, screeningRoom: ScreeningRoom)
```

Comment:
Tutaj poszedłeś w dobrym kierunku decydując się porzucić `startHour: Int, startMinute: Int` - fajnie byłoby to spropagować na resztę kodu :)